tests/bearlib/aspects/DecoratorsTest.py

```diff
@@ -46,22 +62,63 @@ def test_setting_priority(self):
             Setting('remove_unreachable_code', 'False'))
         self.section.append(
             Setting('minimum_clone_tokens', 40))
-        result = self.bear.execute()
-        self.assertEqual([False, 40], result)
+        result = self.list_to_dict(self.bear.execute())
+        self.assertEqual({'remove_unreachable_code': False,
+                          'minimum_clone_tokens': 40,
+                          'use_spaces': True}, result)
 
     def test_partial_mapping(self):
         self.section.aspects = AspectList([
             get_aspect('UnreachableCode')('py'),
         ])
-        result = self.bear.execute()
-        self.assertEqual([True, 10], result)
+        result = self.list_to_dict(self.bear.execute())
+        self.assertEqual({'remove_unreachable_code': True,
```

Comment:
this should be a CONSTANT in the class, used to initialise the expected test results.

```py
EXPECTED = {'remove_unreachable_code': True,
 'minimum_clone_tokens': 10,
 'use_spaces': True,
 }
...

expected = EXPECTED.copy()
expected['blah'] = override
self.assertEqual(expected, results)
```

Reply:
Good idea.
I am changing 'remove_unreachable_code' to False to be consistent with the bear defaults.